compiler/damlc/lib/DA/Cli/Damlc/Packaging.hs

```diff
@@ -121,10 +121,11 @@ createProjectPackageDb opts thisSdkVer deps0 dataDeps = do
                         | LF.PRImport pid <- toListOf packageRefs dalf
                         ]
                 let getUid = getUnitId unitId pkgMap
-                let src = generateSrcPkgFromLf getUid pkgId dalf
+                let src = generateSrcPkgFromLf getUid (Just "Sdk") dalf
                 let templInstSrc =
                         generateTemplateInstancesPkgFromLf
                             getUid
+                            (Just "Sdk")
```

Comment:
Would it be worth unifying all instances of the string "Sdk" into a single named variable?
(I count 10 places in the code base)

Reply:
You're right there.

Reply:
I'll do it in a follow PR.